expfmt/encode.go

```diff
@@ -51,16 +73,44 @@ func Negotiate(h http.Header) Format {
 				return FmtProtoCompact
 			}
 		}
-		// Check for text format.
+		if ac.Type == "text" && ac.SubType == "plain" && (ver == TextVersion || ver == "") {
+			return FmtText
+		}
+	}
+	return FmtText
+}
+
+// NegotiateOpenMetrics works like Negotiate but includes FmtOpenMetrics as an
+// option for the result. Note that this function is experimental and might
+// disappear once FmtOpenMetrics is fully supported and as such may be
+// negotiated by the normal Negotiate function.
+func NegotiateOpenMetrics(h http.Header) Format {
```

Comment:
Can we avoid putting OpenMetrics in the name? NegotiateWithCloser or something maybe.
As-is, you'd expect it to prefer/only-negotiate OM.

Reply:
`NegotiateWithCloser` wouldn't really convey the meaning, either. It would arguably distract from the actual purpose of this function.

`NegotiateIncludingOpenMetrics` perhaps?

I just tried to

Reply:
NegotiateIncludingOpenMetrics sounds good.

Reply:
Done.